tools/cli/src/config.rs

```diff
@@ -0,0 +1,18 @@
+use layer_climb::prelude::*;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct Config {
+    pub chains: ChainConfigs,
+    pub faucet: FaucetConfig,
+}
+#[derive(Debug, Deserialize, Serialize)]
+pub struct ChainConfigs {
```

Comment:
Ah, you already do store these in a config file (.json not .toml).

Let's rethink how to make this a bit more flexible